### DISCUSSION TITLE
日光對大腦有影響

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#878] [General]
https://www.natgeomedia.com/science/article/content-15564.html

每當雨天或是陰天的時候，我都會覺得心情稍微有點差，而原來真的是因為日照影響我們的大腦，那這樣常常下雨的地區民族性是不是會稍微憂鬱消極一點?這篇文章還提及研究者發現能夠操縱老鼠特定SCN神經元，並誘導多巴胺的分泌，若最後真的能夠對人體也有效的話，我想一定能幫助許多不論是有心理疾病、免疫系統出問題或是壓力調控失調的患者等等。

**Comment by peggy0329** [comment]
雨天的時候心情真的會超差！！
我一直以為是我自己的問題哈哈哈，因此之前有查到一篇文章是說天氣冷跟白天較短的冬天容易讓血清素下降，因此會產生「季節性情緒失調」。醫生有建議大家可以規律生活以讓生理時鐘正常與多出門走走，增加照日光的時間。
但我覺得大家都是大學生，真的很難抽出時間多出門走走，更別說平常考試報告真的很難避免熬夜。若如同文章所說的，未來研發出可以誘導人體分泌多巴胺的東西話，感覺會對憂鬱患者或是這種人們常有的季節性情緒失調的小疾病有改善。

**Comment by dinosaur1009** [comment]
這個研究對於平常曬不到太陽的人超有幫助ㄉ 感覺是個很重要的醫療發展

**Comment by HuangHsinTzu** [comment]
之前有統計台灣最多憂鬱症患者的都市就是雨都—基隆!
日光真的很大影響一個人的心情!
陽光明媚的時候心情就會特別好，特別有動力
所以平常還是要多到戶外走走，多接收日光而非藍光阿

**Comment by Pietra1226** [comment]
不知道這是特定對日光才有反應還是其實類日光的光也可以刺激反應，說不定也可以用人為的方式讓反應發生。

**Comment by StevenWanglolz** [comment]
其實現在有很多科技都會誘導多巴胺的分泌了，每天滑的手機就會讓你產生愉悅感。我認為應該要進行Dopamine Detox，然後多出去戶外運動，促進血清素和多巴胺的分泌，改變你大腦對愉悅感獲取方式的認知，才是比較好的方式!

有關Dopamine detox: https://www.medicalnewstoday.com/articles/dopamine-detox#does-it-work

**Comment by Chiway1229** [comment]
我認同樓上的Steven所說的，相對於沉迷於手機
去戶外曬曬太陽更令人感到愉悅

**Comment by bonny123865** [comment]
原來日光會影響心情!長知識了!
希望未來能在醫療應用上用到這個研究結果!

**Comment by Kotori0501** [comment]
下雨天是真的換讓自己的心情感到厭世，而且還會不太想出門！！
這篇文章的實驗意義真的挺有趣的，真的能在研究出來對醫療領域也有一定的幫助，而且像我這種比較內向，整天感到傷心的人，說不定有了這樣的技術下，也變得比較開心一點

**Comment by Zoe06060606** [comment]
看來自律生活真的很重要 有時早點起床去外面曬曬太陽真的能讓人心曠神怡

**Comment by evanchou1204** [comment]
前幾個禮拜一直下雨，真的整個人超煩躁的，還以為是自己的問題QQ

**Comment by ching020721** [comment]
下雨天真的很影響心情，從小就在桃園長大真的很常下雨，一放晴就會很開心

**Comment by Zoe06060606** [comment]
現代的人們 曬太陽的比率越來越高 這項記術對未來真的很重要

**Comment by Evan0979** [comment]
有陽光真的心情會很好!下雨天又陰又濕,一整個就很不舒服

**Comment by CPCpyt** [comment]
我覺得的確是這樣，非常直觀的結果。
因天雨下雨對我整個身心都不好了。
